include/clang/Interpreter/InterOp.h

```diff
@@ -78,10 +82,10 @@ namespace InterOp {
     /// Makes a call to a generic function or method.
     ///\param[in] result - the location where the return result will be placed.
     ///\param[in] args - a pointer to a argument list and argument size.
-    ///\param[in] self - the this pointer of the object.
```

Comment:
The `this pointer` is a term we should keep it.

Reply:
changed to
`param[in] self - the 'this pointer' of the object.`